opac_schema/v1/models.py

```diff
@@ -906,6 +907,45 @@ def legend(self):
     def url(self):
         return self.url_segment
 
+    @property
+    def csl_json(self):
+        """
```

Comment:
@gitnnolabs se isso for uma função e não property, vc poderia passar a URL do site para compor o campo URL

Reply:
Boa

Reply:
Olhando existe CSLs que ignoram o atributo **URL** e forma a **URL** na citação com o **DOI**, exemplo: https://github.com/citation-style-language/styles/blob/master/apa.csl#L1539